src/mongocxx/test/collection.cpp

```diff
@@ -706,6 +741,24 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
         assert_contains_one("bar");
         assert_contains_one("quux");
     }
+
+    SECTION("distinct with collation", "[collection]") {
+        auto doc = document{} << "x"
+                              << "foo" << finalize;
+
+        auto predicate = document{} << "x"
+                                    << "FOO" << finalize;
+
+        auto distinct_opts = options::distinct{}.collation(case_insensitive_collation.view());
+
+        auto distinct_results = coll.distinct("foo", predicate.view(), distinct_opts);
```

Comment:
Yep, you're right, thanks for spotting this.  Pushed a fixup commit addressing this.  This doesn't change correctness of the test (the distinct operation now returns one result with "foo", instead of one result with null), but it clarifies the intent.